keanu-project/src/main/java/io/improbable/keanu/vertices/VertexLabel.java

```diff
@@ -1,30 +1,58 @@
 package io.improbable.keanu.vertices;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
 import java.util.Objects;
 
+import com.google.common.base.Joiner;
+import com.google.common.collect.ImmutableList;
+
 public class VertexLabel {
-    private final String label;
+    private static final char NAMESPACE_SEPARATOR = '.';
```

Comment:
Should we use '/' as Charlie requested?  I don't have strong feelings tbh.

Reply:
It's no big deal since it's only used in `toString`.

I chose `.` because it helps with the intuition that it's similar to a Java namespace.